bgpd/bgp_damp.c

```diff
@@ -582,10 +590,9 @@ void bgp_damp_info_clean(struct bgp_damp_config *bdc, afi_t afi, safi_t safi)
 
 	XFREE(MTYPE_BGP_DAMP_ARRAY, bdc->reuse_list);
 	bdc->reuse_list_size = 0;
-
-	THREAD_OFF(bdc->t_reuse);
```

Comment:
If entering `clear ip bgp dampening` what's going on with a thread then? Shouldn't we cancel it though?

Reply:
Sorry, this was not my change. It somehow came due to merge. I have reverted that change.

Reply:
@sudhanshukumar22 As i stated in my comment on November 24th of last year, there have been a lot of changes in BGP dampening. Please do not just revert things without making sure you are not breaking anything of the previous work.

Reply:
Hi @davischw , please note that I had already taken that change and merged it to my changes. My changes are small.